test/TestPython/test_tutorials.py

```diff
@@ -9,7 +9,7 @@
 
 import sys
 import os
-sys.path.append(os.path.join(os.path.dirname(__file__), r'../Tests/bin/DebugFull/net462/'))
+sys.path.append(os.path.join(os.path.dirname(__file__), r'../Tests/bin/Debug/net472/'))
```

Comment:
This isn't always the correct path.  It depends on the Solution Configuration.

Reply:
@tminka I did replace change with comment.